src/components/Modals/EditCurationAssigneeModal/EditCurationAssigneeModal.tsx

```diff
@@ -62,7 +62,7 @@ export default class EditCurationAssigneeModal extends React.PureComponent<Props
                     value === UNASSIGN_PLACEHOLDER
                       ? `- ${t('curation_page.assign_modal.unassign')} -`
                       : value === address
-                      ? `${profiles[address].avatars[0].name} (${t('collection_row.you')})`
+                      ? `${profiles[address]?.avatars[0].name} (${t('collection_row.you')})`
```

Comment:
If the profile for the given address does not exist,  (${t('collection_row.you')}) will have an empty space in the beginning, is this intentional?

Reply:
Oh yeah, good catch. Instead of that, I just added to show the address and the `(you)` text.